setup.py

```diff
@@ -66,9 +83,9 @@
     # details needed by setup
     install_requires=INSTALL_REQUIRES,
     python_requires=">=3.8",
-    packages=find_packages(include=["bokeh", "bokeh.*"]),
+    packages=find_namespace_packages(include=["bokeh", "bokeh.*"]),
```

Comment:
Subfolders without a `__init__.py` are still considered packages by Python itself.
If all subfolders in a package have `__init__.py` it is OK to only use `find_packages`, however, if that is not the case it is more coherent to use `find_namespace_packages`.

Reply:
I think it is the case that all of our subfolders have `__init__.py`, except the `server/static` ones that have BokehJS inside, are those what you are referring to? I'll admit I don't really understand the difference between these two options, or why one is preferable over the other, currently.

Reply:
Personally, I recommend all users to use `find_namespace_packages()`, because it corresponds to the behaviour of the Python interpreter post-PEP 420.

The 2 functions are kept separated as an attempt to prevent unaware users to misuse the feature (if they don't specify any keyword argument - which is very common - this can cause some havoc, e.g. the `docs` and the `tests` folder might end up in the final wheel by mistake).

In the case of `bokeh`, the `include=` keyword argument is specified, so using `find_namespace_packages()` is safe.

> Are those what you are referring to?

Yes, `server/static` and any eventual subfolder will be considered namespace packages by the Python interpreter.

Projects that use `find_packages` despite having folders without `__init__.py` are starting to see some educational warning since `setuptools` 62.3.0.